packages/dx-react-demos/src/theme-sources/bootstrap3/demo-container.jsx

```diff
@@ -115,9 +115,9 @@ class DemoFrame extends React.PureComponent {
               initialContent={this.markup}
               mountTarget="#mountPoint"
             >
-              {themeName === 'custom' && (
+              {themeName === 'custom' ? (
```

Comment:
Why?

Reply:
Because React doesn't want children will be of type `undefined`.